app/controllers/concerns/api/users_controller.rb

```diff
@@ -49,23 +49,9 @@ def unpublish
       authorize(@user, :unpublish_all_articles?)
 
       target_user = User.find(params[:id].to_i)
-      target_comments = target_user.comments.where(deleted: false)
-
-      # Keep track of affected ids for AuditLog trail
-      article_ids = Article.published.where(user_id: target_user.id).ids
-      comment_ids = target_comments.ids
 
       # Unpublish posts and delete comments w/ boolean attr to allow revert
-      Moderator::UnpublishAllArticlesWorker.perform_async(target_user.id)
-      target_comments.update(deleted: true)
-
-      payload = {
-        action: "api_user_unpublish",
-        target_user_id: target_user.id,
-        target_article_ids: article_ids,
-        target_comment_ids: comment_ids
-      }
-      Audit::Logger.log(:admin_api, @user, payload)
+      Moderator::UnpublishAllArticlesWorker.perform_async(target_user.id, @user.id)
```

Comment:
Just curious: is there a reason this would not create a `Note` as the main app controller does?

Reply:
That a good question :)
Adding creating note from the api will require more coordination, I will discuss and add it in a separate pr (if needed).
Also, I see that there are no `audit_logs` with the corresponding slug in the database, so the action is usually performed via admin action/moderator action panel :thinking: (but it still would be nice to be consistent).